Use tlsh with cmake, missing include "version.h"

When I use the c++ library in my project and include it with `#include tlsh.h` then i have the following error :  
```
/usr/local/include/tlsh.h:66:10 fatal error: version.h: No file found
#include "version.h"
```
It seems like the file /usr/local/include/version.h doesn't exist.
It's generated when i run `make.sh` but is not installed when using `make install` in the `build/release` directory.
Am I doing something wrong during the installation ?

--- a/make.sh
+++ b/make.sh
@@ -17,6 +17,10 @@ if [ $# -eq 1 -a "$1" = "debug" ]; then
   mkdir -p build/debug
   cd build/debug
   cmake -DCMAKE_BUILD_TYPE=Debug ../..
+elif [ $# -eq 1 -a "$1" = "-shared" ]; then
+  mkdir -p build/release
+  cd build/release
+  cmake -DTLSH_SHARED_LIBRARY=1 ../..
 else
   mkdir -p build/release
   cd build/release

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -65,7 +65,10 @@ set_target_properties(tlsh PROPERTIES OUTPUT_NAME tlsh${BUILD_POSTFIX})
 # it was causing problems when compiling / testing tools on Linux
 ##########################
 
-set(TLSH_SHARED_LIBRARY 0)
+if (NOT DEFINED TLSH_SHARED_LIBRARY)
+    set(TLSH_SHARED_LIBRARY 0)
+endif()
+
 if(TLSH_SHARED_LIBRARY EQUAL 1)
     add_library(tlsh_shared SHARED ${TLSH_SRCS})
     set_target_properties(tlsh_shared PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/lib)
@@ -81,3 +84,4 @@ else()
 endif()
 
 install(FILES ../include/tlsh.h DESTINATION include)
+install(FILES ../include/version.h DESTINATION include)

